package.json

```diff
@@ -181,7 +181,7 @@
     "import-meta-resolve": "^2.0.3",
     "ipfs-unixfs": "^7.0.0",
     "ipfs-utils": "^9.0.1",
-    "ipfsd-ctl": "^12.0.0",
+    "ipfsd-ctl": "github:sgtpooki/js-ipfsd-ctl#feature/add-js-kubo-rpc-client",
```

Comment:
will replace with proper version when https://github.com/ipfs/js-ipfsd-ctl/pull/764 is merged and published.